ioSPI/transfer.py

```diff
@@ -0,0 +1,174 @@
+import numba as nb
+import numpy as np
+
+
+def ctf_freqs(N, psize=1.0, d=2):
+    """Make CTF Frequencies.
+
+    Makes 1 or 2 D array of fourier space frequenceis
+
+    Parameters
+    ----------
+    N : int
+        Number of pixels.
+    d : int, 1 or 2
+        Dimension.
+    psize : float
+        Pixel size in Å
+
+    Returns
+    -------
+    freq_1d : numpy.ndarray, shape (N//2,)
+        Frequencies in 1D, with dc at left most.
+    freq_A_2d : numpy.ndarray, shape (N,N)
```

Comment:
`freq_mag_2d`?

Reply:
good point. It's the absolute frequency in 1/Angstroms. I'll rename